detekt-rules-complexity/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/ComplexInterface.kt

```diff
@@ -51,6 +52,9 @@ class ComplexInterface(
     @Configuration("whether private declarations should be included")
     private val includePrivateDeclarations: Boolean by config(defaultValue = false)
 
+    @Configuration("ignore overloaded methods - only count once")
+    private val ignoreOverloaded: Boolean by config(defaultValue = false)
```

Comment:
to be consistent with all other detekt rules

```suggestion
    @Configuration("ignore overridden methods - only count once")
    private val ignoreOverridden: Boolean by config(defaultValue = false)
```

Reply:
These methods are overloaded - not overridden, right? :thinking:
Let me recheck.

Reply:
I got mislead by this comment. The naming is fine. We can keep it.

https://github.com/detekt/detekt/issues/5165#issuecomment-1200915187